library/swift/EngineBuilder.swift

```diff
@@ -211,6 +225,17 @@ open class EngineBuilder: NSObject {
     return self
   }
 
+  /// Set the maximum number of connections to open to a single host.
```

Comment:
same as above - can we add a comment which would explain what the default maximum is (7)?